bin/main.rb

```diff
@@ -1,3 +1,76 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+# puts 'Hello World!'
+module Game
+  def self.start
+    system 'clear'
+    arr = [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9]
+    ]
+
+    puts '+---+---+---+'
+    puts "| #{arr[0][0]} | #{arr[0][1]} | #{arr[0][2]} |"
+    puts '+---+---+---+'
+    puts "| #{arr[1][0]} | #{arr[1][1]} | #{arr[1][2]} |"
+    puts '+---+---+---+'
+    puts "| #{arr[2][0]} | #{arr[2][1]} | #{arr[2][2]} |"
+    puts '+---+---+---+'
+  end
+end
+
+# Players class
+class Player
+  include Game
+  attr_accessor :name
+
+  def initialize
+    puts 'Player 1 name: '
+    @name1 = gets.chomp
+    puts 'Player 2 name: '
+    @name2 = gets.chomp
+  end
+
+  attr_reader :name1, :name2
```

Comment:
attr_reader should be on top and before the #initialize. Kindly remove attr_accessor and write atttr_reader on top.